pygments/lexers/hdl.py

```diff
@@ -316,6 +317,7 @@ class SystemVerilogLexer(RegexLexer):
              Keyword.Type),
             (r'[a-zA-Z_]\w*:(?!:)', Name.Label),
             (r'\$?[a-zA-Z_]\w*', Name),
+            (r'\\(\S+)\s', Name),
```

Comment:
The '\\(\S+)' looks mostly right to me (Verilog only defines behavior on ASCII characters, and I think we will also capture Unicode with this, but that's not important). Is it correct to include the '\s' whitespace character with the identifier? Does it work correctly if you leave this part off and let the whitespace capture rules chomp that part?

Reply:
> Is it correct to include the '\s' whitespace character with the identifier?

No, and neither is the backslash part of the identifier. The identifiers `foo` and `\foo ` (note the space) are completely equivalent in every regard.

> Does it work correctly if you leave this part off and let the whitespace capture rules chomp that part?

I can check that, but what about the backslash?